samplesheets/tests/test_rendering.py

```diff
@@ -202,6 +203,35 @@ def test_get_study_tables_cache(self):
         study_tables = self.tb.get_study_tables(self.study)
         self.assertEqual(study_tables, cache_item.data)
 
+    @override_settings(SHEETS_ENABLE_STUDY_TABLE_CACHE=False)
+    def test_get_study_tables_config(self):
+        """Test get_study_tables() with SHEETS_ENABLE_STUDY_TABLE_CACHE=False"""
+        tables = self.tb.get_study_tables(self.study)
+        t_field = tables['study']['top_header'][0]
+        self.assertEqual(t_field['value'], 'Source')
+
+        sheet_config = self.build_sheet_config(self.investigation)
+        c_field = sheet_config['studies'][str(self.study.sodar_uuid)]['nodes'][
+            0
+        ]
+        self.assertEqual(c_field['header'], 'Source')
```

Comment:
We can omit these lines related to the sheet config, as it is irrelevant for this test.